src/line-helpers.ts

```
@@ -1,3 +1,5 @@
+import { invariant } from "@luxass/utils";
```

ESLint

[error] 'invariant' is defined but never used.

(unused-imports/no-unused-imports)